### ISSUE TITLE
关于VQAv2-to-AOKVQA主实验的复现

### ISSUE THREAD

**Issue by zhengjin11** [KongoCat/Cloud-Edge-Collaborative-Inference-with-LLaMA-Accessory#5]
您的paper让我受益匪浅，但是我在复现VQAv2-to-AOKVQA的实验时，似乎遇到了一些困难，能像您请教一下吗？

## 初始模型的选择 ？

根据原文中4.2小节的句子：“we use the VQAv2-to-AOKVQA datasets for evaluation. Specifically, we adopt VQA-v2 [10] to finetune the pre-trained MLLM, LLaMA-Adapter (7B & 13B). Then, the VQA accuracy results on A-OKVQA [11] under different conditions”。在这个实验上，您先把LLaMA-Adapter (7B & 13B)的模型先在VQA-v2上微调。
请问在VQA-v2上微调之前，模型是从llama-accessory官方发布的checkpoint中下载的吗？
是不是这个文档https://llama2-accessory.readthedocs.io/en/latest/finetune/mm.html# 里的Stage Two Checkpoint

## 训练配置？
在VQA-v2上微调LLaMA-Adapter (13B) 的时候，您使用的脚本是LLaMA2-Accessory官方库下的

accessory/exps/finetune/mm/llavaVqav2_llamaAdapter.sh
accessory/exps/finetune/mm/alpacaLlava_llamaQformerv2Peft_13B.sh
accessory/exps/finetune/mm/alpacaLlava_llamaQformerv2Peft_QF_13B.sh

三者中的哪一个？毕竟这三个脚本训练了模型中不同的部分。
